### ISSUE TITLE
Improve CSS for embedded link previews

### ISSUE THREAD

**Issue by punchagan** [zulip/zulip-mobile#3552]
The CSS for embedded link previews could be improved as noted by Boris in the comment here: https://github.com/zulip/zulip/issues/5885#issuecomment-317125072

The CSS used by the web app for this is here: 
https://github.com/zulip/zulip/blob/master/static/styles/zulip.scss#L1572-L1634

Also, images that are part of the embedded link previews don't seem to be displayed. Already reported in #3191

**Comment by timabbott** [comment]
For some context, this is a feature that's existed for a long time in Zulip (like 2 years) but was disabled by default without hand-editing a setting in /etc/zulip/settings.py for backend implementation quality reasons.

@punchagan has fixed a number of those issues, and we moved it to just a default-off realm setting a few weeks ago.  So making this part of Zulip is newly more at the level of usage of LaTeX rendering or polls than a rare corner case.

**Comment by ishammahajan** [comment]
@timabbott have we enabled this in CZO (doesn't seem to be, because it didn't work when I tried it with some article links)? That'll make debugging a lot easier. If it is not, I'll go make a development server on my machine -- although not sure if it can handle the load :P

**Comment by punchagan** [comment]
Yes, the feature is currently disabled on CZO. I'm not sure if we are yet ready to enable this on CZO - I feel like we should be able to disable previews for GitHub links for instance, before enabling it on CZO. 

You could create a dummy organisation on zulipchat.com and enable the feature there, or ask for a DO droplet to use for your development.

**Comment by ishammahajan** [comment]
Thanks for the information! I'll try to locally setup a server. If that doesn't work out I'll try to go for the methods you mentioned.

**Comment by gnprice** [comment]
@punchagan @timabbott It's been a while since we last looked at this issue; what's the status of this feature at this point on the server side? That'd help us understand how to prioritize it, and also how to go about testing it.
